AdyenComponents/OnlineBanking/OnlineBankingComponent.swift

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
+/// A component that provides a form for Online Banking payment.
+public final class OnlineBankingComponent: PaymentComponent,
+                                           PresentableComponent,
+                                           ComponentLoader,
+                                           PaymentAware {
```

Comment:
I don't think `OnlineBankingComponent ` is a `ComponentLoader `, since it doesn't contain sub components, right?, but it is a `LoadingComponent`

Reply:
yes, you are right.